src/core/types/ui_settings.ts

```diff
@@ -75,6 +75,7 @@ export interface UiSettingsParams<T = unknown> {
   optionLabels?: Record<string, string>;
   /** a flag indicating whether new value applying requires page reloading */
   requiresPageReload?: boolean;
+  preferBrowserSetting?: boolean;
```

Comment:
Docs for this would be useful here

Reply:
agreed, will add.

Reply:
add to https://github.com/opensearch-project/OpenSearch-Dashboards/issues/5831

Reply:
I'm not at all convinced this is a good name, but unfortunately anything "user"-related here would arguably be even more confusing, because it's already in use.